docs/source/examples/hoburgabbeel_ex6_1_output.txt

```diff
@@ -0,0 +1,13 @@
+w = 0.931 * (u_1)**-0.263
+w = 0.814 * (u_1)**-0.0792
+w = 0.996 * (u_1)**-0.436
+w**3.44 = 0.154 * (u_1)**0.584
+        + 0.425 * (u_1)**-2.15
+        + 0.422 * (u_1)**-2.15
+1 = (0.996/w**0.176) * (u_1)**-0.102
+  + (0.973/w**0.114) * (u_1)**-0.015
+  + (0.95/w**0.0937) * (u_1)**0.0161
+
+MA RMS Error:  0.00229216995324
+SMA RMS Error:  2.29966512337e-05
+ISMA RMS Error:  1.33082915919e-06
```

Comment:
@bqpd: should this be automatically generated by running the tests? I don't think it is.

@mjburton11: just to be sure, am I correct that you hand-ran this to generate the output, and then copy/pasted?